tests/components/images.test.ts

```diff
@@ -0,0 +1,178 @@
+    const newLenght = model.getters.getImageIds(sheetId).length;
+    expect(newLenght).toEqual(originalLenght + 1);
+  });
+
+  test("Click on Delete button will delete the image", async () => {
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: "Sheet1" } });
+    await nextTick();
+    expect(fixture.querySelector(".o-figure")).not.toBeNull();
+    await simulateClick(".o-figure");
+    expect(document.activeElement).toBe(fixture.querySelector(".o-figure"));
+    expect(fixture.querySelector(".o-image-menu-item")).not.toBeNull();
+    await simulateClick(".o-image-menu-item");
+    expect(fixture.querySelector(".o-menu")).not.toBeNull();
+    await simulateClick(".o-menu div[data-name='delete']");
+    expect(() => model.getters.getImage(imageId)).toThrow();
+  });
+
+  test("Can copy/paste an image with its context menu", async () => {
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
+    await nextTick();
+    const image = model.getters.getImage(imageId);
+    await simulateClick(".o-figure");
+    await simulateClick(".o-image-menu-item");
+    await simulateClick(".o-menu div[data-name='copy']");
+    paste(model, "A1");
+    expect(model.getters.getImageIds(sheetId).length).toEqual(2);
+    const imageIds = model.getters.getImageIds(sheetId);
+    expect(model.getters.getImage(imageIds[1])).toEqual(image);
+    expect(model.getters.getImage(imageIds[0])).toEqual(image);
+  });
+
+  test("Can cut/paste an image with its context menu", async () => {
+    const sheetId = model.getters.getActiveSheetId();
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
+    await nextTick();
+    const image = model.getters.getImage(imageId);
+    const originalLenght = model.getters.getImageIds(sheetId).length;
+    await simulateClick(".o-figure");
+    await simulateClick(".o-image-menu-item");
+    await simulateClick(".o-menu div[data-name='cut']");
+    paste(model, "A1");
+    const newLenght = model.getters.getImageIds(sheetId).length;
+    expect(newLenght).toEqual(originalLenght);
+    const imageIds = model.getters.getImageIds(sheetId);
+    expect(model.getters.getImage(imageIds[0])).toEqual(image);
+  });
+
+  test("can export a image", async () => {
+    const imageId = "Image1";
+    createImage(model, { figureId: imageId, definition: { sheetId: "Sheet1" } });
+    await nextTick();
+    const data = model.exportData();
+    const activeSheetId = model.getters.getActiveSheetId();
+    const sheet = data.sheets.find((s) => s.id === activeSheetId)!;
+    expect(sheet.figures).toEqual([
+      {
+        id: imageId,
+        tag: "image",
+        height: 380,
+        width: 380,
+        x: 0,
+        y: 0,
+      },
+    ]);
+  });
+
+  test("images have a menu button", async () => {
```

Comment:
Same comment as in the `clipboard_figure.test.ts` file, most of these tests are duplicates of the chart tests, we could do `test.each/describe.each` to test both of theme without duplicating everything.